yotta/data/example-tls-client/README.md

```diff
@@ -105,9 +111,9 @@ To build and run this example the following requirements are necessary:
 
 ## Debugging the TLS connection
 
-If you are experiencing problems with this example, you should first rule out network issues by making sure the [simple HTTP file downloader example](https://github.com/ARMmbed/mbed-example-network-private/tree/maste r/test/helloworld-tcpclient) for the TCP module works as expected. If not, please follow the debug instructions for this example.
+If you are experiencing problems with this example, you should first rule out network issues by making sure the [simple HTTP file downloader example](https://github.com/ARMmbed/mbed-example-network-private/tree/master/test/helloworld-tcpclient) for the TCP module works as expected. If not, please follow the debug instructions below.
```

Comment:
"below" is not what I meant. I meant that if the "simple HTTP file downloader" doesn't work, then the first step is to debug that other examples by following its instructions; the instructions below are only for the case where HTTP (the other example) works but HTTPS (the present example) doesn't.